pkg/controllers/repositoryserver/testutils.go

```diff
@@ -34,6 +34,8 @@ const (
 	DefaultKopiaRepositoryServerAccessPassword = "test1234"
 	DefaultKanisterNamespace                   = "kanister"
 	DefaultKopiaRepositoryServerContainer      = "repo-server-container"
+	defaultRepoConfigFilePath                  = "/tmp/kopia-repo-config"
```

Comment:
lets reuse the `config file path` constant from `command` package.

Reply:
Okay will make the changes